src/EditCommand.php

```diff
@@ -5,6 +5,7 @@
 use Symfony\Component\Process\Process;
 use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputInterface;
```

Comment:
@svpernova09 Is there a reference to this rule. Couldn't find it in PSR-2. I thought it would make sense to order them alphabetically.